<ui version="4.0" >
 <class>CodeMetricsDialog</class>
 <widget class="QDialog" name="CodeMetricsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Code Metrics</string>
  </property>
  <property name="whatsThis" >
   <string>&lt;b>Code Metrics&lt;/b>
&lt;p>This dialog shows some code metrics.&lt;/p></string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTreeWidget" name="resultList" >
     <property name="whatsThis" >
      <string>&lt;b>Code metrics&lt;/b>
&lt;p>This list shows some code metrics.&lt;/p></string>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Start</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>End</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Lines</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Lines of code</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Comments</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Empty</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="summaryList" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis" >
      <string>&lt;b>Summary&lt;/b>
&lt;p>This shows some overall code metrics.&lt;/p></string>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>Summary</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>#</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="checkProgress" >
     <property name="toolTip" >
      <string>Shows the progress of the code metrics action</string>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
